src/main/java/nus/climods/ui/module/SavedModuleCard.java

```diff
@@ -5,7 +5,7 @@
 import javafx.scene.layout.FlowPane;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.Region;
-import nus.climods.model.module.Module;
+import nus.climods.model.module.UserModule;
 import nus.climods.ui.UiPart;
 import nus.climods.ui.common.Pill;
 
```

Comment:
There is a bug with the UI display, the pill is to the left of the module code rather than being on the right